packager/media/event/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 Google LLC. All rights reserved.
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+add_library(media_event STATIC
```

Comment:
Please delete media_event.gyp in this PR.

Reply:
resolved in new version